### DISCUSSION TITLE
Having problem to install Xournal++ in my mac book air

### DISCUSSION THREAD

**Discussion by felipelinar** [xournalpp/xournalpp#3255] [General] [answered]
Hi, I have tried to install Xournal++ in my mac book air and in my mac desktop but I just got a message to ask me to contact you because probably my devices are not match the version for your software.

**Comment by rolandlo** [comment, marked as answer]
What version of Xournal++ did you install? What MacOS version do you use? What was the exact message?

**Reply by felipelinar** [reply]
I use the version of Xournal++ available in the webpage xournalpp-1.1.0-macos.zip, my mcbook air version 10.13.6,
and the message I receive is: Check with the developer to make sure Xournal++ 2 works with this version of macOS. You may need to reinstall the application. Be sure to install any available updates for the application and macOS.

**Reply by rolandlo** [reply]
Xournal++ 1.1.0 is not compatible with MacOS 10.13.6, see https://github.com/xournalpp/xournalpp/pull/2888. You need at least Catalina (MacOS 10.15)

**Reply by felipelinar** [reply]
Is there a version of xournal that mathches my MacOS 10.13.6?

**Reply by rolandlo** [reply]
I think [version 1.0.20](https://github.com/xournalpp/xournalpp/releases/tag/1.0.20) (and some earlier versions) should all be compatible with MacOS 10.13.6. 

**Edit:** corrected the link

**Reply by felipelinar** [reply]
Thanks a lot!
